templates/docker-compose/CeleryExecutor/README.md

```diff
@@ -0,0 +1,29 @@
+# docker-compose-airflow
+Docker Compose Apache Airflow (Official Docker Images) with CeleryExecutor, InitDB and InitUser
+
+Ideal for local development or small scale personal deployments.
+
+Prerequisites: Docker and docker-compose
```

Comment:
Should we move it to "Deploy" section?

Reply:
You mean changing folder name "templates" --> "deploy" ? yeah whatever you guys think appropriate. 
I just created a parent folder, so that going forward people can contribute non docker-compose templates (k8s/helm charts etc.,).
However with this specific config, I just want to reiterate, apart from local development and very small scale deployments, this is not _production ready_ deployment. As docker-compose aimed at single node computing instead of distributed computing.
Users might want to modify this config to fit their ECS/Fargate, K8s and Docker Swarm config etc.,

Reply:
@resdevd I don't know why but I was thinking that this is the main README. I'm ok with this as it is